sweep/sweeper.go

```diff
@@ -282,6 +374,26 @@ func (s *UtxoSweeper) SweepInput(input input.Input) (chan Result, error) {
 	return sweeperInput.resultChan, nil
 }
 
+// validateFeePreference validates a client's fee preference in accordance to
+// the UtxoSweeper's config.
+func (s *UtxoSweeper) validateFeePreference(
+	feePreference FeePreference) (lnwallet.SatPerKWeight, error) {
+
+	feeRate, err := DetermineFeePerKw(s.cfg.FeeEstimator, feePreference)
+	if err != nil {
+		return 0, err
+	}
+
+	if feeRate < s.cfg.MinFeeRate {
```

Comment:
Hmm, now we may reject an input when it is submitted, pushing for example a resolver into an error state. While the next block the estimate may be different and within bounds.

Alternatively, we could just cap the fee with the max.

Reply:
Currently this value is set to the fee floor, which all estimators ensure we don't return a fee rate below. If this was to change though, then the case you bring up could definitely happen. Seems like it might be better to not even make this value configurable then?

Reply:
Ok, so this check is in for manually bumped inputs then?

Reply:
It will be yes.